shared/src/business/entities/contacts/ContactFactory.test.js

```diff
@@ -125,7 +125,7 @@ describe('ContactFactory', () => {
     expect(caseExternal.getFormattedValidationErrors()).toEqual(null);
   });
 
-  it('passes validation when primary contact is defined and everything else is valid on a served case', () => {
+  it('passes validation when petitioner exists and everything else is valid on a served case', () => {
```

Comment:
I think ContactFactory is only used when a case is in `New` status now, so this whole test is irrelevant and can probably be removed.

Reply:
I had a sneaking suspicion. Thanks!